src/screens/Order/OrderUIUtils.res

```diff
@@ -472,3 +474,72 @@ let getOrdersList = async (
 let isNonEmptyValue = value => {
   value->Option.getOr(Dict.make())->Dict.toArray->Array.length > 0
 }
+
+let viewsArray: array<viewTypes> = [All, Succeeded, Failed, Dropoffs, Cancelled]
+
+let getViewsDisplayName = view => {
+  switch view {
+  | All => "All"
+  | Succeeded => "Succeeded"
+  | Failed => "Failed"
+  | Dropoffs => "Dropoffs"
+  | Cancelled => "Cancelled"
+  }
+}
+
+let getAllPaymentsLabel = obj => {
+  open LogicUtils
+  obj
+  ->getDictFromJsonObject
+  ->getDictfromDict("status_with_count")
+  ->Dict.keysToArray
+  ->Array.joinWith(",")
+}
+
+let getViewsLabel = (view, obj) => {
+  switch view {
+  | All => getAllPaymentsLabel(obj)
+  | Succeeded => "succeeded"
+  | Failed => "failed"
+  | Dropoffs => "requires_payment_method"
+  | Cancelled => "cancelled"
+  }
+}
+
+let getViewTypeFromString = view => {
+  switch view {
+  | "succeeded" => Succeeded
+  | "cancelled" => Cancelled
+  | "failed" => Failed
+  | "requires_payment_method" => Dropoffs
+  | _ => All
+  }
+}
+
+let getAllPaymentsCount = obj => {
+  open LogicUtils
+  let countArray =
+    obj
+    ->getDictFromJsonObject
+    ->getDictfromDict("status_with_count")
+    ->Dict.toArray
+    ->Array.map(item => {
+      let (_, value) = item
+      value
+    })
+  countArray->Array.reduce(0, (acc, curr) =>
+    (acc->Int.toFloat +. curr->JSON.Decode.float->Option.getOr(0.0))->Float.toInt
```

Comment:
we have I guess getfloat from json in logic utils check once. if not then add it